### ISSUE TITLE
How to reproduce your results?

### ISSUE THREAD

**Issue by SelahWang** [andrekuros/B-ACE#11]
Thanks for sharing the code and environment for your BVR research! I’m really interested in your work _"Enhancing BVR air combat agent development with attention-driven reinforcement learning"_, but I'm having some trouble getting the reward curves to converge using the provided script: Examples/Tianshou/Example_Tianshou_TasksBased.py.

A couple of quick questions:

- Are the default hyperparameters in this script (learning rate, attention configs, etc.) the same as those used for the results in your paper?

- Do you have any tips or specific "tricks" needed to reproduce the paper's performance?

Any guidance would be super helpful. Thanks!
